galaxykit/registries.py

```diff
@@ -8,7 +8,7 @@ def get_registry_pk(client, name):
     user_url = f"_ui/v1/execution-environments/registries/?name={name}"
     resp = client.get(user_url)
     if resp["data"]:
-        return resp["data"][0]["pk"]
+        return resp["data"][0]["id"]
```

Comment:
this is surprising .. do you also need to change `user_url` to use v3? Or did we really change the value returied by the v1 api?

Reply:
only `repositories/*` endpoints are moved to `/v3/plugin`, `registries` and `remotes` didn't change for now. 

And part of this PR is to unify EE endpoints to have `id, pulp_href, created_at and updated_at`